app/Actions/EntityActions.js

```diff
@@ -93,5 +93,12 @@ export default {
             id,
             view
         });
+    },
+
+    flagResourceNotFound(found) {
```

Comment:
Looks weird to me to have a `ResourceNotFound` method with an argument. It should always be false, shouldn't it?